<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>500</width><height>400</height></rect>
  </property>
  <property name="windowTitle">
   <string>List Operations App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="generateButton">
    <property name="geometry"><rect><x>20</x><y>20</y><width>150</width><height>30</height></rect></property>
    <property name="text"><string>Generate Numbers</string></property>
   </widget>
   <widget class="QLabel" name="label_numbers">
    <property name="geometry"><rect><x>20</x><y>60</y><width>100</width><height>25</height></rect></property>
    <property name="text"><string>Numbers:</string></property>
   </widget>
   <widget class="QTextEdit" name="numbersTextEdit">
    <property name="geometry"><rect><x>120</x><y>60</y><width>350</width><height>50</height></rect></property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry"><rect><x>20</x><y>120</y><width>100</width><height>25</height></rect></property>
    <property name="text"><string>Result:</string></property>
   </widget>
   <widget class="QTextEdit" name="resultTextEdit">
    <property name="geometry"><rect><x>120</x><y>120</y><width>350</width><height>50</height></rect></property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry"><rect><x>20</x><y>180</y><width>100</width><height>25</height></rect></property>
    <property name="text"><string>Status:</string></property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry"><rect><x>120</x><y>180</y><width>350</width><height>25</height></rect></property>
   </widget>
   <widget class="QRadioButton" name="sumRadio">
    <property name="geometry"><rect><x>20</x><y>220</y><width>100</width><height>25</height></rect></property>
    <property name="text"><string>Sum</string></property>
   </widget>
   <widget class="QRadioButton" name="avgRadio">
    <property name="geometry"><rect><x>140</x><y>220</y><width>100</width><height>25</height></rect></property>
    <property name="text"><string>Average</string></property>
   </widget>
   <widget class="QRadioButton" name="maxRadio">
    <property name="geometry"><rect><x>260</x><y>220</y><width>100</width><height>25</height></rect></property>
    <property name="text"><string>Max</string></property>
   </widget>
   <widget class="QRadioButton" name="sortRadio">
    <property name="geometry"><rect><x>380</x><y>220</y><width>100</width><height>25</height></rect></property>
    <property name="text"><string>Sort</string></property>
   </widget>
   <widget class="QComboBox" name="sortComboBox">
    <property name="geometry"><rect><x>20</x><y>260</y><width>150</width><height>25</height></rect></property>
    <item><property name="text"><string>Ascending</string></property></item>
    <item><property name="text"><string>Descending</string></property></item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
